### PR TITLE
Webauthn

### PR DESCRIPTION
**Pull request by BurntVal**
https://github.com/burnt-labs/abstraxion/assets/117188229/3339dfc9-3fb6-4042-9eac-8e17aae56dd0

### CODE REVIEW

File: src/components/AbstraxionSignin/index.tsx [hunk 1]
```diff
@@ -97,6 +99,17 @@ export const AbstraxionSignin = () => {
     suggestAndConnect({ chainInfo: testnetChainInfo, walletType: wallet });
   };
 
+  async function handleWebauthnAuthenticate() {
+    try {
+      await stytchClient.webauthn.authenticate({
+        domain: "localhost",
```
Comment (BurntNerve) [inline src/components/AbstraxionSignin/index.tsx:105]: This is fine for now but we should probably start figuring out how to (and stay with me here) _abstract_ a lot of these config values into environment variables.

**Review by BurntNerve — APPROVED** [review]
Lookin good

**Review by justinbarry — APPROVED** [review]
LGTM

**Review by BurntNerve — APPROVED** [review]
looks great

**Pull request merged**